occurrence-download/src/main/java/org/gbif/occurrence/download/file/simplecsv/SimpleCsvArchiveBuilder.java

```diff
@@ -46,22 +49,23 @@
   //Header file is named '0' to appear first when listing the content of the directory.
   private static final String HEADER_FILE_NAME = "0";
   //String that contains the file HEADER for the simple table format.
-  private static final String HEADER = buildHeader();
-
+  private final String HEADER;
+  
   /**
    * Creates the file HEADER.
    * It was moved to a function because a bug in javac https://bugs.openjdk.java.net/browse/JDK-8077605.
    */
-  private static String buildHeader() {
-    return DownloadTerms.SIMPLE_DOWNLOAD_TERMS.stream()
+  public static SimpleCsvArchiveBuilder withHeader(Set<Term> downloadTermsHeader) {
+    String header =  downloadTermsHeader.stream()
       .map(term -> HiveColumns.columnFor(term).replaceAll("_", ""))
```

Comment:
there's a better way of doing  replaceAlls that avoid re-compiling the regex multiple times, follow suggestions in your IDE